selfhost/compiler.jakt

```diff
@@ -136,14 +136,17 @@ class Compiler {
         .get_file_id_or_register(file_name)
     }
 
-    public function search_for_path(this, anon module_name: String) throws -> Path? {
+    public function search_for_path(this, anon input_module_name: String) throws -> Path? {
+        mut builder = StringBuilder::create()
+        builder.append(get_path_separator())
+        let separator = builder.to_string()
```

Comment:
ouch, but okay

Reply:
I felt the same way. Would it be possible to get a string version of the separator? Or is there a better way to convert u8 to a String?

Reply:
Added FIXME.